doc/OnlineDocs/contributed_packages/community.rst

```diff
@@ -0,0 +1,274 @@
+    a list of all the constraints in the community as well as all the variables contained in those constraints.
+    For the model graph, a node is created for every active constraint in the model, an edge between two
+    constraint nodes is created only if those two constraint equations share a variable, and the
+    weight of each edge is equal to the number of variables the two constraint equations have in common.
+
+    If the user sets `type_of_community_map='v'`, then each entry in the community map (which is a dictionary) contains
+    a list of all the variables in the community as well as all the constraints that contain those variables.
+    For the model graph, a node is created for every variable in the model, an edge between two variable nodes is
+    created only if those two variables occur in the same constraint equation, and the weight of each edge is equal
+    to the number of constraint equations in which the two variables occur together.
+
+    If the user sets `type_of_community_map='b'`, then each entry in the community map (which is a dictionary) is
+    simply all of the nodes in the community but split into a list of constraints and a list of variables.
+    For the model graph, a node is created for every variable and every constraint in the model. An edge is created
+    between a constraint node and a variable node only if the constraint equation contains the variable. (Edges are
+    not drawn between nodes of the same type in a bipartite graph.) And as for the edge weights, the edges in the
+    bipartite graph are unweighted regardless of what the user specifies for the `weighted_graph` parameter. (This is
+    because for our purposes, the number of times a variable appears in a constraint is not particularly
+    useful.)
+
+Weighted Graph/Unweighted Graph
+    The Louvain community detection algorithm takes edge weights into account, so depending on whether the graph is
+    weighted or unweighted, the communities that are found will vary. This can be valuable depending on how
+    the user intends to use the community detection information. For example, if a user plans on feeding that
+    information into an algorithm, the algorithm may be better suited to the communities detected in a weighted
+    graph (or vice versa).
+
+With/Without Objective in the Graph
+    This argument determines whether the objective function(s) will be included when creating the graphical
+    representation of the model and thus whether the objective function(s) will be included in the community map.
+    Some models have an objective function that contains so many of the model variables that it obscures potential
+    communities within a model. Thus, it can be useful to call `detect_communities(model, with_objective=False)`
+    on such a model to see whether isolating the other components of the model provides any new insights.
```

Comment:
Would it make sense to move this description into the function docstring itself, given that we suggest `autofunction` above?

Or, have a condensed version in docstring, and leave this here.

Reply:
Yeah, the intention was for the docstring to be a condensed version of the longer explanation of the community detection in the rst file. But let me know if you're saying I should modify the docstring based on the descriptions in the rst file or if you're saying the docstring is fine as it is.